### ISSUE TITLE
SSUI tests fail on [DetailsServiceCatalogsView] display timeout

### ISSUE THREAD

**Issue by istein1** [ManageIQ/integration_tests#8098]
Tests [1] are failing on CFME-5.9.6.0/RHV-4.2.7-2 with this error, which seem to be a general issue, and not RHV provider specific:

request = <FixtureRequest for <Function 'test_quota_tagging_infra_via_services[group-rhv_cfme_integration-max_memory-ViaSSUI]'>>
appliance = IPAppliance.from_json('{"container": null, "ui_port": 443, "ssh_port": 22, "op...s_dev": false, "version": "5.9.6.0", "db_host": null, "browser_steal": false}')
provider = RHEVMProvider(endpoints={'default': <cfme.infrastructure.provider.rhevm.RHEVME...ne='default', start_ip='10.46.16.76', end_ip='10.46.16.76', provider_data=None)
setup_provider = None, admin_email = None
context = <class 'cfme.utils.appliance.implementations.ssui.ViaSSUI'>
set_entity_quota_tag = None, custom_prov_data = {'hardware': {'memory': '4096'}}
prov_data = {'catalog': {'catalog_name': {'name': 'golden_mixed_virtio_template_full'}, 'vm_name': 'test-quota-qvjl'}, 'environment': {'automatic_placement': True}, 'hardware': {'memory': '4096'}, 'network': {'vlan': partial_match('ovirtmgmt')}}
catalog_item = RHVCatalogItem(name='test_j6hoyZogp0', prov_data={'environment': {'automatic_p...escription=u'my dialog 790i82dvUj'), domain='ManageIQ (Locked)', provider=None)

    @pytest.mark.rhv2
    @pytest.mark.meta(blockers=[BZ(1633540, forced_streams=['5.10'],
        unblock=lambda provider: not provider.one_of(RHEVMProvider))])
    @pytest.mark.parametrize('context', [ViaSSUI, ViaUI])
    # Here set_entity_quota_tag is used for setting the tag value.
    # Here custom_prov_data is used to provide the value fo the catalog item to be created.
    @pytest.mark.parametrize(
        ['set_entity_quota_tag', 'custom_prov_data'],
        [
            [('Quota - Max Memory *', '1GB'), {'hardware': {'memory': '4096'}}],
            [('Quota - Max Storage *', '10GB'), {}],
            [('Quota - Max CPUs *', '1'), {'hardware': {'num_sockets': '8'}}]
        ],
        indirect=['set_entity_quota_tag'],
        ids=['max_memory', 'max_storage', 'max_cpu']
    )
    def test_quota_tagging_infra_via_services(request, appliance, provider, setup_provider, admin_email,
                                              context, set_entity_quota_tag, custom_prov_data,
                                              prov_data, catalog_item):
        """This test case verifies the quota tagging is working correctly for the infra providers."""
    
        prov_data.update(custom_prov_data)
        with appliance.context.use(context):
            service_catalogs = ServiceCatalogs(appliance, catalog_item.catalog, catalog_item.name)
            if context is ViaSSUI:
> service_catalogs.add_to_shopping_cart()

cfme/tests/infrastructure/test_quota_tagging.py:193: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../.cfme/lib/python2.7/site-packages/sentaku/context.py:131: in __call__
    return bound_method(*k, **kw)
cfme/services/service_catalogs/ssui.py:67: in add_to_shopping_cart
    view = navigate_to(self, 'Details')
../.cfme/lib/python2.7/site-packages/navmazing/__init__.py:123: in navigate
    return nav(cls_or_obj, self, self.logger).go(0, *args, **kwargs)
cfme/utils/appliance/implementations/ssui.py:180: in go
    message="Waiting for view [{}] to display".format(view.__class__.__name__)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

func = <function <lambda> at 0x12ef7e60>, func_args = [], func_kwargs = {}
logger = <cfme.utils.log.TraceLogger object at 0x31f6850>
kwargs = {'message': 'Waiting for view [DetailsServiceCatalogsView] to display', 'num_sec': 10}
st_time = 1541612568.856045, total_time = 8, num_sec = 10, expo = False
f_code = <code object <lambda> at 0xaf4b930, file "/var/lib/jenkins/workspace/cfme-5.9-...-dev/integration_tests/cfme/utils/appliance/implementations/ssui.py", line 179>
line_no = 179
filename = '/var/lib/jenkins/workspace/cfme-5.9-rhv-4.2-integration-test-dev/integration_tests/cfme/utils/appliance/implementations/ssui.py'

    def wait_for(func, func_args=[], func_kwargs={}, logger=None, **kwargs):

. . .

> raise TimedOutError("Could not do {} at {}:{} in time".format(message, filename, line_no))
E TimedOutError: Could not do Waiting for view [DetailsServiceCatalogsView] to display at /var/lib/jenkins/workspace/cfme-5.9-rhv-4.2-integration-test-dev/integration_tests/cfme/utils/appliance/implementations/ssui.py:179 in time

../.cfme/lib/python2.7/site-packages/wait_for/__init__.py:180: TimedOutError

[1]
test_quota_tagging_infra_via_services[group-rhv_cfme_integration-max_memory-ViaSSUI]         
test_quota_tagging_infra_via_services[group-rhv_cfme_integration-max_storage-ViaSSUI]         
test_quota_tagging_infra_via_services[group-rhv_cfme_integration-max_cpu-ViaSSUI]         
test_quota_tagging_infra_via_services[user-rhv_cfme_integration-max_memory-ViaSSUI]         
test_quota_tagging_infra_via_services[user-rhv_cfme_integration-max_storage-ViaSSUI]         
test_quota_tagging_infra_via_services[user-rhv_cfme_integration-max_cpu-ViaSSUI]         
test_tenant_quota_enforce_via_service_infra[rhv_cfme_integration-max_cpu-ViaSSUI]         
test_tenant_quota_enforce_via_service_infra[rhv_cfme_integration-max_storage-ViaSSUI]         
test_tenant_quota_enforce_via_service_infra[rhv_cfme_integration-max_memory-ViaSSUI]         
test_tenant_quota_enforce_via_service_infra[rhv_cfme_integration-max_vms-ViaSSUI]

**Comment by mshriver** [comment]
@sshveta has been assigned JIRA issue 10478, and I'm going to close this issue as it is an effective duplicate of #7960, where we have been tracking these failures.  #8091 will include some fixes for these catalog views.
